RELEASING.md

```diff
@@ -0,0 +1,16 @@
+Releasing
+=========
+
+1. Auto-generate the changelog for the latest version using [conventional-changelog-cli]
+    e.g. `conventional-changelog -p angular`
+2. Inspect the autogenerated changelog, make modifications as needed, and update [CHANGELOG.md].
+3. Update the [README.md] "Installation" section with the new version of the library
+4. Update the version in [gradle-mvn-push.gradle] with the new version of the library
+5. Create a pull request with the changes with the title "chore(Release): <version_number>"
+6. Once the pull request has been merged, create a git tag and push it (Travis will handle publishing to Maven)
+7. Create a GitHub release with the tag
+
+[conventional-changelog-cli]: https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-cli
+[gradle-mvn-push.gradle]: https://github.com/googlemaps/android-maps-utils/blob/master/gradle-mvn-push.gradle#L65
```

Comment:
You can similarly do:

~~~
[gradle-mvn-push.gradle]: gradle-mvn-push.gradle#L65
~~~

...here and Android Studio resolves the local file link, but doesn't seem to resolve the link to the line number. From some quick Googling I don't know if GitHub would be smart enough to resolve the line number in the shortened format.

Reply:
Good to know - updating. I'm not seeing Android Studio handle markdown previews for me though. Are you using a plugin?

Reply:
I'm not seeing Markdown rendering previews either right now (although I believe it did at some point? Or maybe that was IntelliJ proper...). But if I Ctrl-click (Windows) on the text in the text editor it will still take me to the file still.

Here are the plugins I have installed and how my editor looks:

![image](https://user-images.githubusercontent.com/928045/75293302-ea6e7b80-57f3-11ea-951c-1bd77bb4fced.png)